fpga_interchange/nextpnr.py

```diff
@@ -56,11 +56,14 @@ def str(self, s, comment=None):
             print('str |{}| {}'.format(s, comment), file=self.f)
 
     def str_id(self, s):
-        index = self.const_ids.get_index(s)
+        if s == ('', ):
```

Comment:
for the sake of my limited python knowledge; what is the reason for comparing against a single-valued tuple here?